### PR TITLE
chore(i3s examples): add i3s colorization by attributes

### PR DESCRIPTION
**Pull request by maxkuznetsov-actionengine**

### CODE REVIEW

**Review by belom88 — CHANGES_REQUESTED** [review]
This is too large for an example.
I'd just have a dropdown with attributes to colorize. We don't need to have histogram, set of icons, picking and so on.
Could we make colorization and i3s removal in different PRs?

**Comment by dsavinov-actionengine** [comment]
> This is too large for an example. I'd just have a dropdown with attributes to colorize. We don't need to have histogram, set of icons, picking and so on. Could we make colorization and i3s removal in different PRs?

1. For the UI, would you both @maxkuznetsov-actionengine @belom88 agree with this?
image

2.  Splitting in 2 different PRs - ok.

**Comment by belom88** [comment]
> > This is too large for an example. I'd just have a dropdown with attributes to colorize. We don't need to have histogram, set of icons, picking and so on. Could we make colorization and i3s removal in different PRs?
> 
> 1. For the UI, would you both @maxkuznetsov-actionengine @belom88 agree with this?
>    image
> 
> For Colorize Mode: None/Replace/Multiply For Colorize Attribute: None/Selected Attribute None is default for both
> 
> 2. Splitting in 2 different PRs - ok.

I suggest swiping controls. User selects from top to down direction. 1st - select attribute, 2nd - select mode (if needed)

**Review by ibgreen — APPROVED** [review]

File: examples/website/i3s-colorization-by-attributes/src/types.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export type COLOR = [number, number, number, number];
```
Comment (belom88) [inline examples/website/i3s-colorization-by-attributes/src/types.ts:1]: CAPS is for constants

```suggestion
export type Color = [number, number, number, number];
```

File: examples/website/i3s-colorization-by-attributes/src/types.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export type ColorsByAttribute = {
+  attributeName: string;
+  minValue: number;
+  maxValue: number;
+  minColor: COLOR;
```
Comment (belom88) [inline examples/website/i3s-colorization-by-attributes/src/types.ts:7]: There is a `Color` type in @deck.gl

**Review by belom88 — APPROVED** [review]

File: examples/website/i3s-colorization-by-attributes/src/types.ts (hunk 1 above)
Comment (maxkuznetsov-actionengine) [reply examples/website/i3s-colorization-by-attributes/src/types.ts:1]: Removed type definition

File: examples/website/i3s-colorization-by-attributes/src/types.ts (hunk 2 above)
Comment (maxkuznetsov-actionengine) [reply examples/website/i3s-colorization-by-attributes/src/types.ts:7]: added import from deck

**Pull request merged**